glslang/MachineIndependent/preprocessor/PpContext.h

```diff
@@ -430,16 +431,30 @@ class TPpContext {
         TInputScanner* input;
     };
 
-    // Holds a string that can be tokenized via the tInput interface.
+    // Holds a reference to included file data, as well as a 
+    // prologue and an epilogue string. This can be scanned using the tInput
+    // interface and acts as a single source string.
     class TokenizableString : public tInput {
```

Comment:
The class name should be changed.   It's more than just a string now.
Suggest TokenizableIncludeFile or TokenizableIncludedShaderSource